frontend/packages/integration-tests-cypress/tests/crud/namespace-crud.cyspec.ts

```diff
@@ -0,0 +1,57 @@
+import { testName } from '../../utils/common';
+import { listPage } from '../../views/list-page';
+import { modal } from '../../views/modal';
+
+describe('Namespace', () => {
+  before(() => {
+    cy.login();
+    cy.createProject(testName);
+  });
+  after(() => {
+    cy.deleteProject(testName);
+    cy.logout();
+  });
+
+  const newName = `${testName}-ns`;
+
+  it('Create, Retrieve (list), Update (labels)', () => {
```

Comment:
Let's establish best practices for our test descriptions. They should start with a lowercase verb like `creates a namespace`. For example:

```suggestion
  it('creates a namespace and updates its labels', () => {
```

(Let's make sure all of the tests follow the pattern.)

Reply:
ok, fixed